src/trust-region/trust-region.jl

```diff
@@ -47,7 +47,7 @@ function ratio(nlp :: AbstractNLPModel, f :: Float64, f_trial :: Float64, Δm ::
   pred < 0 || throw(TrustRegionException(@sprintf("Nonnegative predicted reduction: pred = %8.1e", pred)))
 
   ared = f_trial - f + max(1.0, absf) * 10.0 * ϵ
-  if (Δm < 1.0e+4 * ϵ) || (abs(ared) < 1.0e+4 * ϵ * absf)
+  if (abs(Δm) < 1.0e+4 * ϵ) || (abs(ared) < 1.0e+4 * ϵ * absf)
```

Comment:
I thought this was trying to check for small delta m, because delta m is negative.